deploy-service/common/src/main/java/com/pinterest/deployservice/bean/Role.java

```diff
@@ -30,13 +30,16 @@
  *      plus the ability specify new OPERATORS and ADMINs for said environment.
  *      When a new environment is created the creating user is the designated the
  *      first ADMIN.
+ * SYSTEMADMIN:
+ *      System level administrator. This role is for system-wide use.
  */
 public enum Role {
     READER(0),
     PINGER(1),
     PUBLISHER(1),
     OPERATOR(10),
-    ADMIN(20);
+    ADMIN(20),
+    SYSTEMADMIN(100);
```

Comment:
may not be used if we can check SYSTEM

Reply:
Will this role be available in UI to select?

Reply:
Can you share some more info about your question? I can add additional code to support adding a role, if needed.

Reply:
I will check for: API, UI.

Reply:
2 to check:
POST /v1/envs/{envName}/user_roles
POST /v1/envs/{envName}/group_role